Subject: x86_64: fiuxp pt_reqs leftovers

The hpet_rtc_interrupt handler still uses pt_regs. Fix it.

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
Signed-off-by: Ingo Molnar <mingo@elte.hu>

---
 arch/x86_64/kernel/hpet.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.22/arch/x86_64/kernel/hpet.c
===================================================================
--- linux-2.6.22.orig/arch/x86_64/kernel/hpet.c	2007-07-24 08:56:54.000000000 +0200
+++ linux-2.6.22/arch/x86_64/kernel/hpet.c	2007-07-24 08:56:55.000000000 +0200
@@ -439,7 +439,7 @@ int hpet_rtc_dropped_irq(void)
 	return 1;
 }
 
-irqreturn_t hpet_rtc_interrupt(int irq, void *dev_id, struct pt_regs *regs)
+irqreturn_t hpet_rtc_interrupt(int irq, void *dev_id)
 {
 	struct rtc_time curr_time;
 	unsigned long rtc_int_flag = 0;
